resources/facts.txt

```diff
@@ -68,3 +68,33 @@ Several ships have wrecked on Disappointment Island.
 The Eiffel Tower is repainted every seven years.
 In 1992, a shipping crate containing 29,000 rubber ducks, beavers, turtles, and frogs fell overboard. They washed up around the world for the next 20 years, and have been used to track ocean currents.
 A group of lemurs is called a conspiracy.
+The first person convicted of speeding was going eight mph.
+"New car smell" is the scent of dozens of chemicals.
+Some sea snakes can breathe through their skin.
+The heads on Easter Island have bodies.
+The moon has moonquakes.
+Goosebumps are meant to ward off predators.
+Humans are the only animals that blush.
+The wood frog can hold its pee for up to eight months.
+Your nostrils work one at a time.
+Only two mammals like spicy food: humans and the tree shrew.
+Rabbits can't puke.
+The "M's" in M&Ms stand for "Mars" and "Murrie."
+Copper door knobs are self-disinfecting.
+Cotton candy was invented by a dentist.
+The English word with the most definitions is "set."
+The dot over the lower case "i" or "j" is known as a "tittle."
+Chewing gum boosts concentration.
+The longest wedding veil was the same length as 63.5 football fields.
+The unicorn is the national animal of Scotland.
+Bees sometimes sting other bees.
+Kids ask 300 questions a day.
+The total weight of ants on earth once equaled the total weight of people.
+"E" is the most common letter and appears in 11 percent of all english words.
+Humans are just one of the estimated 8.7 million species on Earth.
+The Comic Sans font came from an actual comic book.
+Napoleon was once attacked by thousands of rabbits.
+Some planets produce diamond rain.
+Sharks can live for five centuries.
```

Comment:
Change to "Some scientists estimate that the Greenland Shark can live for up to 500 years.